exporters/common/src/main/java/io/opentelemetry/exporter/internal/http/HttpExporter.java

```diff
@@ -77,7 +78,7 @@ public CompletableResultCode export(T exportRequest, int numItems) {
           try {
             body = httpResponse.responseBody();
           } catch (IOException ex) {
-            throw new IllegalStateException(ex);
+            throw new UncheckedIOException(ex);
```

Comment:
We should probably just log a warning here instead of throwing. At this point, we've already concluded that the export has failed. The body is only used in the log message.

Reply:
I agree, just changed it